### DISCUSSION TITLE
How to model an individual with mild renal impairment?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1695] [Questions & Problems]
Hi!
I found that the built-in CKD module in PK-Sim were only for individuals with eGFR below 60mL/min/1.73 m2. What parameters should I change if I want to set up an individual with mild renal impairment, i.e. eGFR between 60-90mL/min/1.73 m2? Where can I find the change of physiological parameters in this population group?
Best regards,

**Comment by SCZ1174** [comment]
Requesting an answer... 
It does seem a bit cumbersome if the only way to change the parameters manually.

**Comment by prvmalik** [comment]
Not meaningfully parameterized or different from healthy
